Use GitHub mirror for FreeBSD bindists instead of FreeBSD distcache

arrowd opened the pull request:
I have double-checked each bindist's checksum, everything's matches.

borsboom commented:
I'm fine with this change, but I'm curious about the motivation.  In https://github.com/commercialhaskell/stackage-content/pull/82, we stopped using the Github mirrors for official bindists, since Github release assets don't actually perform all that well, so since then I've been sticking with the distcache assets for new FreeBSD bindists as well. Are there speed or reliability concerns about the FreeBSD distcache?

arrowd commented:
> I'm fine with this change, but I'm curious about the motivation. In #82, we stopped using the Github mirrors for official bindists, since Github release assets don't actually perform all that well, so since then I've been sticking with the distcache assets for new FreeBSD bindists as well. Are there speed or reliability concerns about the FreeBSD distcache?

Oh, I see. In this case, I reverted all the changes except for GHC 8.6.5, because I removed it from the distcache already, so it needs to be pulled from GitHub.
